x-pack/plugins/security_solution/public/resolver/view/panel.tsx

```diff
@@ -46,15 +46,21 @@ import { CrumbInfo } from './panels/panel_content_utilities';
  *
  * @returns {JSX.Element} The "right" table content to show based on the query params as described above
  */
-const PanelContent = memo(function PanelContent() {
+const PanelContent = memo(function PanelContent({
+  documentLocation,
+}: {
+  documentLocation: string;
+}) {
   const history = useHistory();
   const urlSearch = history.location.search;
   const dispatch = useResolverDispatch();
-
+  const crumbIdKey = `${documentLocation}CrumbId`;
+  const crumbEventKey = `${documentLocation}CrumbEvent`;
   const { timestamp } = useContext(SideEffectContext);
-  const queryParams: CrumbInfo = useMemo(() => {
-    return { crumbId: '', crumbEvent: '', ...querystring.parse(urlSearch.slice(1)) };
+  const queryParams: any = useMemo(() => {
+    return { uniqueCrumbId: '', uniqueCrumbEvent: '', ...querystring.parse(urlSearch.slice(1)) };
```

Comment:
:information_source: [documentLocation + 'crumbId']: '' would let you have any number of them as long as documentLocation is stable and unique across pageloads, right?

Reply:
oh wait nvm, I see you're already doing that above